docs/conf.py

```diff
@@ -216,39 +223,74 @@ def fetch_parsec_version():
 
 # -- Options for LaTeX output ------------------------------------------
 
+latex_engine = "lualatex"
+
 latex_elements = {
+    #  Additional stuff for the LaTeX preamble.
+    "preamble": "\n".join(
+        [
+            # Use firecode in codeblocks.
+            r"\usepackage{lstfiracode}",
+        ]
+    ),
+    # Customize used font in latex
+    "fontpkg": "\n".join(
+        [
+            r"\usepackage{fontspec}",
+            r"\setmainfont{roboto}",
+            r"\setsansfont{roboto}",
+            r"\setmonofont{firacode}",
+        ]
+    ),
     # The paper size ('letterpaper' or 'a4paper').
-    # 'papersize': 'letterpaper',
+    "papersize": "a4paper",
     #  The font size ('10pt', '11pt' or '12pt').
-    # 'pointsize': '10pt',
-    #  Additional stuff for the LaTeX preamble.
-    # 'preamble': '',
+    "pointsize": "12pt",
+    "printindex": r"\footnotesize\raggedright\printindex",
+    "fncychap": r"\usepackage[Sonny]{fncychap}",
+    "babel": r"\usepackage{babel}",
 }
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass
 # [howto/manual]).
-latex_documents = [("index", "parsec.tex", "Parsec Documentation", "Emmanuel Leblond", "manual")]
+latex_documents = [
+    ("index", "parsec.tex", "Parsec Full Documentation", "dev-parsec+fullguide@scille.fr", "howto"),
```

Comment:
IMHO, `Full` is a little redundant here.
```suggestion
    ("index", "parsec.tex", "Parsec Documentation", "dev-parsec+fullguide@scille.fr", "howto"),
```

Reply:
done